docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM conda/miniconda3:latest
+MAINTAINER EiffL <francois.lanusse@gmail.com>
+
+# Installs additional software dependencies
+RUN apt-get update  && apt-get install -y libplot-dev plotutils git build-essential
+
+# Installs conda dependencies
+RUN conda update -y conda && conda install -y numba=0.40 pyyaml scipy
+
+# Installs all of Erin's stack
+RUN pip install git+https://github.com/esheldon/fitsio@unicode \
```

Comment:
the latest version of the code has been updated to work with the latest fitsio release on conda